ports/espressif/esp-idf-config/sdkconfig-esp32c3.defaults

```diff
@@ -46,4 +46,7 @@ CONFIG_NEWLIB_NANO_FORMAT=y
 
 # end of Component config
 
+# Workaround for https://github.com/espressif/esp-idf/issues/14456
```

Comment:
This comment will disappear when these files are automatically updated using `make update-all-sdkconfig`

Reply:
I removed it. Reinforcing my dislike of Kconfig.